calculation-service/pom.xml

```diff
@@ -36,6 +36,23 @@
             <groupId>org.projectlombok</groupId>
             <artifactId>lombok</artifactId>
         </dependency>
+
+        <dependency>
+            <groupId>org.springframework.cloud</groupId>
+            <artifactId>spring-cloud-starter-openfeign</artifactId>
```

Comment:
Do we need this dependency?
Shouldn't it be inherited transitively from the contracts?